src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -216,9 +220,33 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
+    <EmbeddedResource Include="Resources\Strings.de.resx" />
+    <EmbeddedResource Include="Resources\Strings.es.resx" />
+    <EmbeddedResource Include="Resources\Strings.fr.resx" />
+    <EmbeddedResource Include="Resources\Strings.it.resx" />
+    <EmbeddedResource Include="Resources\Strings.ja.resx" />
+    <EmbeddedResource Include="Resources\Strings.ko.resx" />
+    <EmbeddedResource Include="Resources\Strings.ru.resx" />
+    <EmbeddedResource Include="Resources\Strings.zh-Hans.resx" />
+    <EmbeddedResource Include="Resources\Strings.zh-Hant.resx" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="project.json" />
   </ItemGroup>
   <ItemGroup>
-     <None Include="project.json" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.de.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.es.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.fr.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.it.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.ja.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.ko.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.ru.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.zh-Hans.xlf" />
+    <XliffResource Include="MultilingualResources\Microsoft.CSharp.zh-Hant.xlf" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <Import Project="$(MSBuildExtensionsPath)\Microsoft\Multilingual App Toolkit\Microsoft.Multilingual.ResxResources.targets" Label="MultilingualAppToolkit" Condition="Exists('$(MSBuildExtensionsPath)\Microsoft\Multilingual App Toolkit\Microsoft.Multilingual.ResxResources.targets')" />
+  <Target Name="MATPrerequisite" BeforeTargets="PrepareForBuild" Condition="!Exists('$(MSBuildExtensionsPath)\Microsoft\Multilingual App Toolkit\Microsoft.Multilingual.ResxResources.targets')" Label="MultilingualAppToolkit">
+    <Warning Text="$(MSBuildProjectFile) is Multilingual build enabled, but the Multilingual App Toolkit is unavailable during the build. If building with Visual Studio, please check to ensure that toolkit is properly installed." />
```

Comment:
We should definitely avoid this warning as there will be builds, i.e. x-plat builds, that will not even have the ability to fix this warning. What happens if the tools are not actually present?